### ISSUE TITLE
Interface support cannot handle multiple labels

### ISSUE THREAD

**Issue by kraney** [neo4j-graphql/neo4j-graphql-js#298]
The queries generated for fields that reference GraphQL interfaces arbitrarily take the first label on a node, and thus only work when the node has no more than one label.

A node with multiple labels (e.g. n:LogicalEntity:Item) when returned on a field referencing an interface may get this error nondeterministically, depending on the order of the list returned by a call to labels():

"Abstract type Area must resolve to an Object type at runtime for field GeneralArea.includes with value { name: \"SomeItem\", FRAGMENT_TYPE: \"LogicalEntity\" }, received \"LogicalEntity\". Either the Area type should provide a \"resolveType\" function or each possible type should provide an \"isTypeOf\" function.",

E.g.
```
interface A {
}

type B implements A {
}

type C implements A {
}

interface X {
}

type Y implements X {
}

type Z implements X {
}

type Container {
  exes: [X!]! @relation(name: "CONTAINS", direction: "OUT")
  ays: [A!]! @relation(name: "CONTAINS", direction: "OUT")
}
```
...and where the nodes are labeled :A:B, :A:C, :X:Y, or :X:Z. Possibly with other labels as well.

_Originally posted by @kraney in https://github.com/neo4j-graphql/neo4j-graphql-js/pull/270#issuecomment-522205876_

**Comment by kraney** [comment]
Something of a workaround here, although it requires a naming convention where extra labels beyond the concrete type must start with underscore:
https://github.com/kraney/neo4j-graphql-js/commit/73483377eb100a701957f6b39a5ab354770a21ad

**Comment by flazoon** [comment]
This should be fixed by https://github.com/neo4j-graphql/neo4j-graphql-js/pull/336

**Comment by johnymontana** [comment]
Closing as this was fixed in #336, please reopen if you're still seeing this behavior.
